<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOGIN FORM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Register">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>UserName</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="UserName"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Password"/>
           </item>
           <item>
            <widget class="QLineEdit" name="Email"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Register_User">
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableWidget" name="ReadTable">
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>UserName</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Password</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Email</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ReadData">
       <property name="text">
        <string>Read Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="del_label">
       <property name="text">
        <string>Select User ID  to Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="del_choose"/>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteUser">
       <property name="text">
        <string>Delete User</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
